tests/test_linestring.py

```diff
@@ -216,6 +216,12 @@ def test_numpy_coords(self):
         la = np.asarray(line.coords)
         assert_array_equal(la, expected)
 
+        # Check empty
+        line = LineString([])
+        la = np.asarray(line.coords)
+
+        self.assertEqual(la.shape[0], 0)
```

Comment:
@chadwhawkins would you be willing to make this assertion into a separate test function? Like a `test_empty_linestring_coords`? Test isolation is super important to us. We don't want a failure at line 217 to block executing the not related assertion at line 223.

Reply:
I admit that shapely is not the best example historically of isolated test cases, but we're aspiring to do better :smile: